scout/server/blueprints/omics_variants/controllers.py

```diff
@@ -48,3 +58,62 @@ def decorate_omics_variant(store, institute_obj, case_obj, omics_variant_obj):
     update_variant_case_panels(case_obj, omics_variant_obj)
 
     return omics_variant_obj
+
+
+def download_omics_variants(case_obj: dict, variant_objs: CursorType):
+    """Download omics variants in a csv file."""
+
+    def generate(header, lines):
+        yield header + "\n"
+        for line in lines:
+            yield line + "\n"
+
+    DOCUMENT_HEADER = [
+        "Gene",
+        "Gene annotation",
+        "category",
+        "sub-category",
+        "Potential impact",
+        "Value",
+        "P-value",
+        "Fold-change",
```

Comment:
👍 I'm sure there will be more other columns of interest as well that are not on the web page, but this is a good start of the process!